src/uucore_procs/src/lib.rs

```diff
@@ -24,6 +24,9 @@ pub fn main(_args: TokenStream, stream: TokenStream) -> TokenStream {
 
     let new = quote!(
         pub fn uumain(args: impl uucore::Args) -> i32 {
+            #[cfg(unix)]
+            uucore::signals::disable_rust_signal_handlers().expect("Disabling rust signal handlers failed");
```

Comment:
please a comment explaining the why

Reply:
I added a comment as requested, however it is redundant with the comment above the function definition.